### ISSUE TITLE
Custom Invocation

### ISSUE THREAD

**Issue by FatalEagle** [DMOJ/site#77]
Allow users to submit code to a specific judge to test out the speed of execution on that particular judge. Will work nicely with anonymous submissions and hacks.

**Comment by quantum5** [comment]
Implies that judges are unequal.

**Comment by FatalEagle** [comment]
Yeah, hkccc15s3 TLEs on Herculaneum but works on swords because of difference in I/O.

**Comment by quantum5** [comment]
Users are not really supposed to know this. Otherwise religious behaviours will result.

**Comment by Xyene** [comment]
...and that's why you mention it on a *public* tracker. Nice!

**Comment by FatalEagle** [comment]
I think some of the users have already experienced this (submitting on stnbd4). Well, for most problems a correct algorithm will pass on any judge it is assigned to. Of course, there is the issue of incorrect algorithms passing, but the issue isn't too much.

**Comment by quantum5** [comment]
Admins should be responsible to make the differences in judges transparent.

**Comment by Xyene** [comment]
Transparent would involve having a cluster of same-spec servers, which our budget simply does not allow.

**Comment by quantum5** [comment]
Which is why we try to make some sensitive problems run only a selected judges. Something is rather wrong when users end up religiously choosing the "fastest" judges, defeating the purpose of multiple judges in the first place.

**Comment by Xyene** [comment]
@quantum5 See my comment on #79; your comment makes sense there rather than here.
